### PR TITLE
upgrade from 3.11 failed to upgrade es settings from configmap

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-ansible/pull/11917

### CODE REVIEW

**Comment by mburke5678** [comment]
@richm PTAL

**Comment by mburke5678** [comment]
Blocked by https://bugzilla.redhat.com/show_bug.cgi?id=1752532

**Review by richm — APPROVED** [review]
/lgtm

**Comment by mburke5678** [comment]
@anpingli Now that the associated BZ is approved, can you take a look to see if I documented the fix correctly?

**Review by anpingli — COMMENTED** [review]
LGTM

**Comment by mburke5678** [comment]
@anpingli Thanks for the review. 

Is this for the upgrade from 3.10 to 3.11 only? I think I read the BZ wrong. I might need to rebase against 3.10.

**Comment by anpingli** [comment]
It should be 3.11.  We use 3.11 playbook to upgrade from 3.10 to 3.11

**Comment by mburke5678** [comment]
@anpingli 

> We use 3.11 playbook to upgrade from 3.10 to 3.11

Thank you. That makes sense.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL.

**Review by bobfuru — COMMENTED** [review]
LGTM w/one question for clarification.

File: upgrading/automated_upgrades.adoc
Comment (richm) [reply upgrading/automated_upgrades.adoc]: no - no specific to enterprise - the feature is available for all products

**Pull request merged**
